core/selection.ts

```diff
@@ -283,25 +314,24 @@ class Selection {
     return range;
   }
 
-  rangeToNative(range) {
-    const indexes = range.collapsed
```

Comment:
Range#collapsed was removed in a refactor: https://github.com/quilljs/quill/commit/d47f03c342e5280fa2be6d6e3da43adbac47d8c5.